conda/environments/cuspatial_dev_cuda11.1.yml

```diff
@@ -7,11 +7,11 @@ channels:
 dependencies:
   - clang=8.0.1
   - clang-tools=8.0.1
-  - cudf=0.20.*
+  - cudf=21.06.*
```

Comment:
We don't actually build / support CUDA 11.1, so we may want to drop this file altogether?